src/test/infrastructure/controller/TcfApiControllerTest.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-template-curly-in-string */
+import {TestableTcfApiInitializer} from '../../testable/infrastructure/bootstrap/TestableTcfApiInitializer'
+import {expect} from 'chai'
+import itParam from 'mocha-param'
+
+describe('TcfApiController should', () => {
+  describe('version behaviour', () => {
+    const commandList = [
+      'addEventListener',
+      'removeEventListener',
+      'getTCData',
+      'getVendorList'
+    ]
+    const commandListOk = commandList.filter(
+      item => item !== 'removeEventListener'
+    )
+
+    itParam(
+      'for command ${name} If the version is 2, callback shall be invoked with  true for the success parameter and TC data parameter should be returned',
```

Comment:
here should be for command ${value} instead of name, right?

Reply:
done.